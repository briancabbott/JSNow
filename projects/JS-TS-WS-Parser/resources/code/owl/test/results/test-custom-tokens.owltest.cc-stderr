parse error: invalid token '1'
error: 'test executable' exited with status 255
