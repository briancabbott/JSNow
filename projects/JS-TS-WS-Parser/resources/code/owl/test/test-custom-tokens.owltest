1 2342 1.0j other
---#using owl.v2

.token integer "1" "2342"
.token imaginary "1.0j"
.token other

input = (integer | imaginary | other)+
