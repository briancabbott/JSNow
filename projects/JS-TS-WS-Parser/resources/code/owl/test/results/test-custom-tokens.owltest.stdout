. 1 2342 1.0j other
  input------------
